backend/Tournament/managers.py

```diff
@@ -0,0 +1,7 @@
+from .models import Tournament, TournamentPlayer
+
+class TournamentSetupManager:
```

Comment:
Eh we are again changing the logic, now we have different managers for different states of the tournament? Why not just one manager for the Tournament, no matter what the state of the tournament is.